internal/exec/stages/mount/mount.go

```diff
@@ -72,12 +74,31 @@ func (s stage) Run(config types.Config) error {
 	return nil
 }
 
+// returns an error if any element of path is a symlink
+func checkForSymlinks(path string) error {
```

Comment:
Minor, though I'd name this `validateMountPointPath()` or something to make clearer what the intent is.

Reply:
I'd like to keep validate to mean "checks that can be done on the config without needing to be on the host". It's also not doing other validation. Maybe a comment at the caller's site instead?

Reply:
Though now with the change above, it's no longer specifically checking for symlinks, it's just checking for "non-dirs". So maybe something like `checkDirPath`?